node/utils/simulation.ts

```diff
@@ -41,13 +43,23 @@ const getMarketingData = (segment?: SegmentData) => {
 export const getSimulationPayloadsByItem = (
   item: ItemWithSimulationInput,
   segment?: SegmentData,
-  regionId?: string
+  regionId?: string,
+  changeSeller?: Boolean
```

Comment:
Same feedback here, I'd like this variable to be more precisely. `changeSeller` makes me think that we will always be changing the seller, but looking at the code it shows that it's conditional.